parquet-column/src/main/java/org/apache/parquet/internal/column/columnindex/OffsetIndexBuilder.java

```diff
@@ -80,11 +90,22 @@ public void add(int compressedPageSize, long rowCount) {
     @Override
     public void add(long offset, int compressedPageSize, long rowCount) {
     }
+
+    @Override
+    public OffsetIndex build() {
```

Comment:
Why add these two changes? Does this perform as same as before?

Reply:
If I don't, `TestParquetFileWriter.testAlignmentWithPadding` and `TestParquetFileWriter.testAlignmentWithNoPadding` will fail in the Github CI (but not on my laptop) due to writing offset index from NoOpOffsetIndexBuilder, which is an undefined behavior.